### PR TITLE
CASMINST-5424: Upgrade Procedure Fixes

### PR DESCRIPTION
**Pull request by ekamin-hpe**
# Description

This pull request addresses the simpler items to fix in the upgrade procedure. See commit messages for details.

# Checklist Before Merging

- [X] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [X] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [X] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by jld-dehn-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Comment by rustydb** [comment]
@ekamin-hpe the merge button is green for me, does this need any further SME approval/reviews or soak/test time?

**Comment by ekamin-hpe** [comment]
> the merge button is green for me, does this need any further SME approval/reviews or soak/test time?

Let's wait while I open a PR to release/1.3 please.

**Pull request merged**
